src/web/templates/admin/index/config_modal.html

```diff
@@ -10,157 +10,119 @@ <h2 class="modal-title">
-                            ) }}
-                        </div>
-                        <div class="col-12 col-md-6 mb-3">
-                            {{ admin_macros.select_input(
-                                name='console_color',
-                                label=_('Use colors on the console:'),
-                                help_text=_('Using colors helps in finding quickly important information on the console.'),
-                                select_options=console_color_options,
-                                data=data, errors=errors
-                            ) }}
-                        </div>
-                        <div class="col-12 col-md-6 mb-3">
-                            {{ admin_macros.select_input(
-                                name='console_show_date',
-                                label=_('Show the date and time on the console:'),
-                                help_text=_('Showing the date and time on the console may help in identifying slow operations on the console (the date and time are always printed to file if file output is enabled).'),
-                                select_options=console_show_date_options,
-                                data=data, errors=errors
-                            ) }}
-                        </div>
-                        <div class="col-12 col-md-6 mb-3">
-                            {{ admin_macros.select_input(
-                                name='console_show_level',
-                                label=_('Show the logging level on the console:'),
-                                help_text=_('Colorblind people can print the logging level to distinguish important information on the console.'),
-                                select_options=console_show_level_options,
-                                data=data, errors=errors
-                            ) }}
-                        </div>
+                    <div class="col-12 col-md-6 mt-2 d-flex flex-column gap-2">
+                        {{ admin_macros.switch_input(
+                            name='console_show_level',
+                            label='',
+                            placeholder=_('Show the logging level'),
+                            help_text='',
+                            info=_('Colorblind people can print the logging level to distinguish important information on the console.'),
+                            data=data, errors=errors
+                        ) }}
+                        {{ admin_macros.switch_input(
+                            name='console_color',
+                            label='',
+                            help_text='',
+                            placeholder=_('Use log-level specific colors'),
+                            info=_('Using colors helps in finding quickly important information on the console.'),
+                            data=data, errors=errors
+                        ) }}
```

Comment:
```suggestion
                        {{ admin_macros.switch_input(
                            name='console_color',
                            label='',
                            help_text='',
                            placeholder=_('Use log-level specific colors'),
                            info=_('Using colors helps in finding quickly important information on the console.'),
                            data=data, errors=errors
                        ) }}
                        {{ admin_macros.switch_input(
                            name='console_show_level',
                            label='',
                            placeholder=_('Show the logging level'),
                            help_text='',
                            info=_('Colorblind people can print the logging level to distinguish important information on the console.'),
                            data=data, errors=errors
                        ) }}
```
A bit nicer this way I think.